test/db/analysis/x86_64

```diff
@@ -2309,7 +2309,7 @@ aae
 pd 1 @ 0x08048432
 EOF
 EXPECT=<<EOF
-[36m|[0m           [32m0x08048432[0m      [37mlea  [36m eax[0m,[36m [0m[33mstr.Hello_PIC[0m[36m[0m[0m
+[36m|[0m           [32m0x08048432[0m      [37mlea[0m[37m   [0m[36meax[0m[37m, [33mstr.Hello_PIC[0m[0m[0m
```

Comment:
The space characters are now classified as separator tokens. Since each token gets is own color assigned they now get "colored" in white (`37m`). Before the refactor, spaces where simply "colored" like the string before them and the color reset escape string came after them.